docs/_posts/murat-gunay/2022-02-23-bert_sequence_classifier_trec_coarse_pipeline_en.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: model
+title: Classifier Pipeline to Identify Trec Data Classes
+author: John Snow Labs
+name: bert_sequence_classifier_trec_coarse_pipeline
+date: 2022-02-23
+tags: [bert_sequence, trec, coarse, bert, en, open_source]
+task: Text Classification
+language: en
+edition: Spark NLP 3.4.0
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained pipeline is built on the top of [bert_sequence_classifier_trec_coarse_en](https://nlp.johnsnowlabs.com/2021/11/06/bert_sequence_classifier_trec_coarse_en.html).
```

Comment:
I removed here the ** , it was just for you to take a look, not for including the symbols in the description.

Reply:
we can still have it since UI accepts markdown script, if you want.